### PR TITLE
Add project_urls to setup

### PR DESCRIPTION
**Pull request by jarrodmillman**
These are used by PyPI and other services.

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,12 @@
 MAINTAINER_EMAIL = 'stefan@sun.ac.za'
 URL = 'https://scikit-image.org'
 LICENSE = 'Modified BSD'
-DOWNLOAD_URL = 'https://github.com/scikit-image/scikit-image'
+DOWNLOAD_URL = 'https://scikit-image.org/download.html'
+PROJECT_URLS = {
+    "Bug Tracker": 'https://github.com/scikit-image/scikit-image/issues',
```
Comment (hmaarrfk) [inline setup.py]: please use HTTPS

Comment (hmaarrfk) [reply setup.py]: sorry, for the Documentation website.

Comment (jarrodmillman) [reply setup.py]: Done.

**Comment by jni** [comment]
Thanks @jarrodmillman!

**Comment by soupault** [comment]
@meeseeksdev backport to v0.15.x

**Pull request merged**